05 - coder-files/src/App.jsx

```diff
@@ -1,16 +1,25 @@
+import Card from "./components/Card";
+
+const DATA = [
```

Comment:
this DATA should go below the component for readability, great job on including it in the element where it belongs.Also find a good name , DATA is not descriptive